### DISCUSSION TITLE
How to send a value as None in extracting `web::Json<MyForm>`

### DISCUSSION THREAD

**Discussion by cubicYYY** [actix/actix-web#3223] [Q&A] [answered]
```
pub struct MyForm {
    pub latitude: Option<Option<f64>>,
}
```
What I mean: "`None` as 'not provided', `Some(None)` as `a Null value to proceed`".  
However when I trying to specify it by passing `{"latitude":null}` to the API endpoint, it has the same effect as not provided (yes, this makes sense when it comes to JavaScript).  
My question is, how should I make it possible? Some special ways to pass the args in the JSON form or something to do in the rust code?

**Comment by robjtede** [comment, marked as answer]
The easiest solution I can advise is to look at the `js-option` crate in order to achieve the tri-state semantics.

**Reply by cubicYYY** [reply]
That is a neat solution. Thank you!
